examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<ul><li>Natural questions:</li><li class="ql-indent-1">What solver is used below 1000 dimension?</li><li class="ql-indent-1">Which parameters have any affect in this case?</li><li>Let's suggest plausible alternatives to <code>steps = 100</code>. Would it make sense to try 50 or even 10? And also, speculate on when someone would want to go to 200 or 1000?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>